<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pickRaces</class>
 <widget class="QDialog" name="pickRaces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pickRaces</string>
  </property>
  <widget class="QTableWidget" name="racesTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>520</width>
     <height>350</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Location</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Laps</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>To Tournament</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pick Races</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
